src/Generator.h

```diff
@@ -2334,9 +2337,6 @@ class GeneratorOutputBase : public GIOBase {
 
     // }@
 
-#undef HALIDE_OUTPUT_FORWARD
```

Comment:
Why this change? Is the macro used later?

Reply:
Oh, sorry, drive-by change -- apparently a holdover from way back when there were multiple forwarding macros, just dead code now